runtime/jvmti/jvmtiStackFrame.c

```diff
@@ -214,14 +215,28 @@ jvmtiGetThreadListStackTraces(jvmtiEnv* env,
 					goto deallocate;
 				}
 
-				threadObject = *((j9object_t*) thread);
-				targetThread = J9VMJAVALANGTHREAD_THREADREF(currentThread, threadObject);
+				threadObject = J9_JNI_UNWRAP_REFERENCE(thread);
+				if (IS_VIRTUAL_THREAD(currentThread, threadObject)) {
+					j9object_t carrierThread = (j9object_t)J9VMJAVALANGVIRTUALTHREAD_CARRIERTHREAD(currentThread, threadObject);
+					if (NULL != carrierThread) {
+						targetThread = J9VMJAVALANGTHREAD_THREADREF(currentThread, carrierThread);
+					} else {
+						continuationObject = (j9object_t)J9VMJAVALANGVIRTUALTHREAD_CONT(currentThread, threadObject);
+					}
+				} else {
+					targetThread = J9VMJAVALANGTHREAD_THREADREF(currentThread, threadObject);
+					if (NULL != targetThread && targetThread->threadObject != threadObject) {
+						continuationObject = (j9object_t)J9VMJAVALANGVIRTUALTHREAD_CONT(currentThread, targetThread->threadObject);
+						targetThread = NULL;
+					}
+				}
 				if (targetThread == NULL) {
 					currentStackInfo->frame_count = 0;
 				} else {
 					rc = jvmtiInternalGetStackTrace(env,
 					                                currentThread,
 					                                targetThread,
+													continuationObject,
```

Comment:
- indent misaligned.
- Have we updated `jvmtiInternalGetStackTrace` to handle the `continuationObject`?

Reply:
> Have we updated jvmtiInternalGetStackTrace to handle the continuationObject?
Yes, it's part of https://github.com/eclipse-openj9/openj9/pull/15788